apps/web/src/pages/auth/components/QuestionnaireForm.tsx

```diff
@@ -81,6 +83,17 @@ export function QuestionnaireForm() {
   const onCreateOrganization = async (data: IOrganizationCreateForm) => {
     if (!data?.organizationName) return;
 
+    const val = parse(data?.domain as string);
+
+    if (data.domain && !val.isIcann) {
+      setError('domain', {
+        type: 'custom',
+        message: 'Please make sure you specified a valid domain',
```

Comment:
To match the tone of other error messages we have in the app, let's revise this as follows:

```suggestion
        message: 'Please provide a valid domain',
```